custom/icds_reports/templates/icds_reports/icds_app/mobile/additional-filter.html

```diff
@@ -0,0 +1,38 @@
+{% load staticfiles %}
+<div ng-show="$ctrl.filters.indexOf('gender') === -1" class="col-xs-10 float-unset no-gutters mb-5 mx-auto">
```

Comment:
having now seen this more than once maybe make it a variable or function on the controller so it's more readable?